clang/test/dpct/test_api_level/DplExtrasMemory/api_test11.cu

```diff
@@ -1,11 +1,11 @@
 // UNSUPPORTED: cuda-8.0
 // UNSUPPORTED: v8.0
-// RUN: dpct --format-range=none  --usm-level=none  --use-custom-helper=api -out-root %T/DplExtrasMemory/api_test11_out %s --cuda-include-path="%cuda-path/include" -- -x cuda --cuda-host-only -std=c++17
+// RUN: dpct --format-range=none  --usm-level=none  --use-custom-helper=api -out-root %T/DplExtrasMemory/api_test11_out %s --cuda-include-path="%cuda-path/include" -- -x cuda --cuda-host-only -std=c++17 -fsized-deallocation
 // RUN: grep "IsCalled" %T/DplExtrasMemory/api_test11_out/MainSourceFiles.yaml | wc -l > %T/DplExtrasMemory/api_test11_out/count.txt
 // RUN: FileCheck --input-file %T/DplExtrasMemory/api_test11_out/count.txt --match-full-lines %s
 // RUN: rm -rf %T/DplExtrasMemory/api_test11_out
 
```

Comment:
What's the justification for changing the check result from 37 to 38?  I didn't see anything in the commit message explaining why the test result should change.

Reply:
The original PR (https://github.com/oneapi-src/SYCLomatic/pull/319) branched from https://github.com/oneapi-src/SYCLomatic/commit/a969b474c844441344c9f0a1a0bb9dfb729bdce4, and then when it was merged and rebased, the changes were applied to a history which included the commit https://github.com/oneapi-src/SYCLomatic/commit/9ce844bcea9d00ea4cceeb0e4e15ef2e44d8f8a3, which changed the counts.